rules/rbe_repo.bzl

```diff
@@ -898,8 +910,11 @@ def rbe_autoconfig(
     if bazel_rc_version and not bazel_version:
         fail("bazel_rc_version can only be used with bazel_version.")
 
-    if not create_java_configs and java_home != None:
-        fail("java_home should not be set when create_java_configs is false.")
+    if not create_java_configs and (java_home != None or detect_java_home):
```

Comment:
if not create_java_configs and (java_home or detect_java_home):

Reply:
fixed